src/main/java/org/takes/facets/fork/MediaType.java

```diff
@@ -42,6 +43,11 @@
 final class MediaType implements Comparable<MediaType> {
 
     /**
+     * Media type priority clean pattern.
+     */
+    private static final Pattern CLEANER = Pattern.compile("[^0-9\\.]");
```

Comment:
@Happy-Neko This is a bad name. Please check: http://www.yegor256.com/2015/03/09/objects-end-with-er.html